_includes/_scripts/algolia.html

```diff
@@ -0,0 +1,10 @@
+<!-- Algolia search for doc -->
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script type="text/javascript"> docsearch({
```

Comment:
```suggestion
<script type="text/javascript"> if ($("#doc-search-bar").length) docsearch({
```

Reply:
@Philippus 
Thank you for your review. Sorry, maybe I made mistakes when I test this locally.
I updated it. 

One more question if we will choose this approach, no-need to divide algolia's assets into other files. So, should I revert directory and file structure change?